src/classes.py

```diff
@@ -1,20 +1,68 @@
-from typing import Optional
+from typing import Optional, Union
+
 from pydantic import BaseModel
 
 # NOTE
-# 1. blob_id default to 0 to allow for initial type-hint enforcement before blob added to db
+# 1. blob_hash default to 0 to allow for initial type-hint enforcement before blob added to db
 
-
-class BlobInfo(BaseModel):
-    blob_type:str
-    metadata:dict
-    blob_b64s:str
+# Blob Types 
 
 class Fruit(BaseModel):
     entry_id:Optional[int]
+    blob_type:str 
     fruit_name:str 
     fruit_color: Optional[str]
-    blob_id:str = None
+    blob_hash:str 
+
+class Chess(BaseModel):
+    entry_id: Optional[int]
+    blob_type:str = 'chess'
+
+class Youtube(BaseModel):
+    entry_id: Optional[int]
+    blob_type:str = 'youtube'
+    blob_hash:str  
+    photo_id: str 
+    channel: str
+    category: str
+    title:str 
+
+# Blob_type Record 
+Blob_Type = {'fruit':Fruit, 'chess':Chess, 'youtube':Youtube}
+blob_classes = Union[Fruit, Chess, Youtube]
+
+# endpoint inputs 
+
+class BlobGetData(BaseModel):
```

Comment:
use "schema" instead of "data" in these names.

Reply:
done